modules/backend/widgets/lists/partials/_list_pagination.htm

```diff
@@ -34,7 +34,8 @@
                 name="page"
                 class="form-control input-sm custom-select select-no-search"
                 data-request="<?= $this->getEventHandler('onPaginate') ?>"
-                data-load-indicator="<?= e(trans('backend::lang.list.loading')) ?>">
+                data-load-indicator="<?= e(trans('backend::lang.list.loading')) ?>"
+                autocomplete="off">
```

Comment:
@mjauvin Is there any particular reason for the `autocomplete="off"` definitions on the pagination?

Reply:
@bennothommo that was not part of my PR, not sure where that's coming from... !?

Reply:
@mjauvin It looks like another PRs changes have merged in with this one. I'll need you to revert them or take them out of this PR, unfortunately.

Reply:
@bennothommo Not sure why it now shows all those commits, it's fine in my repository when I diff against origin:develop.

Reply:
@mjauvin Yeah I just had a look and it looked fine too. Weird. Would you mind re-creating the PR, and closing this one, just to see if that fixes it?

Reply:
@bennothommo I can, but before I do this, can you do a rebate to see if that fixes it?

Reply:
@mjauvin Not sure if I can do that on an external repo. I can give it a try.

Reply:
@mjauvin No luck, unfortunately.